$NetBSD: patch-ag,v 1.2 2011/09/12 16:35:43 taca Exp $

* Avoid to use timelocal.pl which cause obsolete warning with perl 5.14.
* Make sure to use 4-digit year.

--- dateconv.pl.orig	1998-05-29 19:04:32.000000000 +0000
+++ dateconv.pl
@@ -47,7 +47,8 @@ sub lstime_to_standard
 }
 
 
-require 'timelocal.pl';
+use Time::Local;
+
 package dateconv;
 
 # Use timelocal rather than gmtime.
@@ -149,5 +150,5 @@ sub main'time_to_standard
 
 	local( $sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst ) =
 		 gmtime( $time );
-	return sprintf( "%2d $months[ $mon + 1 ] %2d %02d:%02d", $mday, $year, $hour, $min );
+	return sprintf( "%2d $months[ $mon + 1 ] %4d %02d:%02d", $mday, $year + 1900, $hour, $min );
 }
